python/lsst/pipe/base/graph.py

```diff
@@ -173,6 +173,21 @@ def quanta(self):
             for quantum in taskNodes.quanta:
                 yield taskDef, quantum
 
+    def quanta_as_qgraph(self):
```

Comment:
For consistency with other methods I think it better to use camelCase for method name (`quantaAsQgraph`).